### PR TITLE
Update installation.rst

### PR DESCRIPTION
**Pull request by mblue9**
Hello, maybe you don't want this but am just suggesting it as it is what worked smoothly for me i.e. using python venv instead of virtualenv. With the virtualenv method I ended up in Stack Overflow trying to fix a similar situation to [this](https://stackoverflow.com/questions/31133050/virtualenv-command-not-found).

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -5,21 +5,21 @@ Installation
...
+For a traditional Python installation of Planemo, first set up a virtual environment
+for ``planemo`` (this example creates a new one in ``planemo``) and then
 install with ``pip``. Planemo requires pip 7.0 or newer.
 
 ::
 
-    $ virtualenv .venv; . .venv/bin/activate
-    $ pip install "pip>=7" # Upgrade pip if needed.
+    $ python3 -m venv planemo
+    $ source planemo/bin/activate
```
Comment (nsoranzo) [inline docs/installation.rst:15]: `.` is POSIX and more portable than `source` (which is not available in e.g. dash)

```suggestion
    $ . planemo/bin/activate
```

File: docs/installation.rst [hunk 2]
```diff
@@ -5,21 +5,21 @@ Installation
...
     $ pip install planemo
 
 When installed this way, planemo can be upgraded as follows:
 
 ::
 
-    $ . .venv/bin/activate
+    $ source planemo/bin/activate
```
Comment (nsoranzo) [inline docs/installation.rst:22]:
```suggestion
    $ . planemo/bin/activate
```

**Comment by mblue9** [comment]
Ok great, thanks for the info @nsoranzo !

**Comment by bgruening** [comment]
Thanks @mblue9!

**Comment by mblue9** [comment]
Great! (thanks @rlupat for the tip about python venv)

**Pull request merged**